src/modules/Backoffice/DocumentTable.tsx

```diff
@@ -0,0 +1,272 @@
+  text-align: center;
+  color: #666666;
+  font-size: 14px;
+`;
+
+const MobileOnly = styled.div`
+  display: none;
+  @media (max-width: 768px) {
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+  }
+`;
+
+const DocGrid = styled.div`
+  display: grid;
+  grid-template-columns: 100px 1fr;
+  border: 1px solid #e0e0e0;
+  border-radius: 8px;
+
+  @media (min-width: 769px) {
+    grid-template-columns: repeat(4, 1fr);
+  }
+`;
+
+const LabelCell = styled.div`
+  background: #f8f8f8;
+  font-weight: 600;
+  color: #333;
+  padding: 12px 16px;
+  border-right: 2px solid #e0e0e0;
+  border-bottom: 1px solid #e0e0e0;
+`;
+
+const ValueCell = styled.div`
+  padding: 12px 16px;
+  border-bottom: 1px solid #e0e0e0;
+  word-break: break-word;
+`;
+
+export function DocumentTable({
+  documents,
+  onEdit,
+  onDelete,
+  selectedCategory,
+}: DocumentTableProps) {
+  const { isDesktop, isWidescreen } = useBreakpoint();
+  if (documents.length === 0) {
+    const hasCalendarDownloadAll =
+      selectedCategory === 'Calendar' &&
+      documents.some(
+        (doc) => doc.category === 'Calendar' && doc.is_download_all,
+      );
+
+    if (selectedCategory !== 'Calendar' || !hasCalendarDownloadAll) {
+      return (
+        <EmptyState>
+          No documents found. Click &quot;Add New Document&quot; to create one.
+        </EmptyState>
+      );
+    } else {
+      return null;
+    }
+  }
+  const showDateColumn = documents.some((doc) => !!doc.date);
+  return (
+    <TableContainer>
+      <DesktopOnly>
+        <Table>
+          <Thead>
+            <tr>
+              <Th scope="col">Title</Th>
+              <Th scope="col">URL</Th>
+              {showDateColumn && <Th scope="col">Meeting Date</Th>}
+              <Th scope="col">Actions</Th>
+            </tr>
+          </Thead>
+          <tbody>
+            {documents.map((doc) => (
+              <Tr key={doc.id}>
+                <Td width={isWidescreen ? (isDesktop ? '22%' : '21%') : '35%'}>
+                  {doc.title}
+                </Td>
+                <Td>
+                  <Typography color="gold">
+                    <StyledLink
+                      href={doc.url}
+                      aria-label={`Open ${doc.title} in new tab`}
+                    >
+                      {doc.url.length > 40
+                        ? isDesktop
+                          ? `${doc.url.slice(0, 20)}…`
+                          : `${doc.url.slice(0, 40)}…`
+                        : doc.url}
+                    </StyledLink>
+                  </Typography>
+                </Td>
+                {showDateColumn && (
+                  <Td>
+                    {doc.date ? (
+                      <time dateTime={doc.date}>{formatDate(doc.date)}</time>
+                    ) : (
+                      '—'
+                    )}
+                  </Td>
+                )}
+                <Td>
+                  <ActionButtons>
+                    <ActionButton
```

Comment:
By default, buttons default to type='submit' when inside a form; I understand this isn't a form, but consider setting type="button" to buttons that don't actually submit everything

Reply:
safety thing for accessibility